optimizely/event/user_event_factory.py

```diff
@@ -20,7 +20,7 @@ class UserEventFactory(object):
 
     @classmethod
     def create_impression_event(
-        cls, project_config, activated_experiment, variation_id, user_id, user_attributes,
+        cls, project_config, activated_experiment, variation_id, user_id, user_attributes, flag_key, flag_type
```

Comment:
This new field is not documented. Also, would be good to rethink order of these params.